<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavdataDebugWindow</class>
 <widget class="QWidget" name="NavdataDebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>554</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>554</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Navdata</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelAltitude">
           <property name="text">
            <string>AltitudeOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAdcDataFrame">
           <property name="text">
            <string>AdcDataFrameOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCks">
           <property name="text">
            <string>CksOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDemo">
           <property name="text">
            <string>DemoOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelEulerAngles">
           <property name="text">
            <string>EulerAnglesOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelGames">
           <property name="text">
            <string>GamesOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelGyrosOffsets">
           <property name="text">
            <string>GyrosOffsetsOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHdvideoStream">
           <property name="text">
            <string>HdvideoStreamOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelKalmanPressure">
           <property name="text">
            <string>KalmanPressureOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMagneto">
           <property name="text">
            <string>MagnetoOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPhysMeasures">
           <property name="text">
            <string>PhysMeasuresOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPressureRaw">
           <property name="text">
            <string>PressureRawOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPwm">
           <property name="text">
            <string>PwmOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelRawMeasures">
           <property name="text">
            <string>RawMeasuresOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelRcReferences">
           <property name="text">
            <string>RcReferencesOption</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditAltitude">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAcdDataFrame">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditCks">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDemo">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEulerAngles">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGames">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGyrosOffsets">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditHdvideoStream">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditKalmanPressure">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMagneto">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPhysMeasures">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPressureRaw">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPwm">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditRawMeasures">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditRcReferences">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="bottomMargin">
          <number>48</number>
         </property>
         <item>
          <widget class="QLabel" name="labelReferences">
           <property name="text">
            <string>ReferencesOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTime">
           <property name="text">
            <string>TimeOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTrackersSend">
           <property name="text">
            <string>TrackersSendOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTrims">
           <property name="text">
            <string>TrimsOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVideoStream">
           <property name="text">
            <string>VideoStreamOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVisionDetect">
           <property name="text">
            <string>VisionDetectOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVisionOf">
           <property name="text">
            <string>VisionOfOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVision">
           <property name="text">
            <string>VisionOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVisionPerf">
           <property name="text">
            <string>VisionPerfOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVisionRaw">
           <property name="text">
            <string>VisionRawOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelWatchdog">
           <property name="text">
            <string>WatchdogOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelWifi">
           <property name="text">
            <string>WifiOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelWind">
           <property name="text">
            <string>WindOption</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelZimmu3000">
           <property name="text">
            <string>Zimmu3000Option</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>42</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEditReferences">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTime">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTrackersSend">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTrims">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVideoStream">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVisionDetect">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVisionOf">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVision">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVisionPerf">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVisionRaw">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditWatchdog">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditWifi">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditWind">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditZimmu3000">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textNavdataDebug">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonNavdataDebugClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNavdataShow">
        <property name="text">
         <string>Show</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonNavdataDebugClear</sender>
   <signal>clicked()</signal>
   <receiver>textNavdataDebug</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
